servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -243,7 +243,22 @@ public String createBlock(long userId, long blockId, int tierAlias, long initial
   public BlockWriter createBlockRemote(long userId, long blockId, int tierAlias, long initialBytes)
       throws FileDoesNotExistException, IOException {
     BlockStoreLocation loc = BlockStoreLocation.anyDirInTier(tierAlias);
-    mBlockStore.createBlockMeta(userId, blockId, loc, initialBytes);
+    TempBlockMeta createdBlock = mBlockStore.createBlockMeta(userId, blockId, loc, initialBytes);
```

Comment:
Currently it is a little bit confusing because both `createBlockRemote` and `getTempBlockWriterRemote` returns `BlockWriter`. I think we may change this method `createBlockRemote` to only create a block---without returning a `BlockWriter`. And we always use `getTempBlockWriterRemote` to return the `BlockWriter`